docs/pages/includes/tpm-joining-background.mdx

```diff
@@ -0,0 +1,31 @@
+The `tpm` join method is a secure way for Bots and Agents to authenticate with
+a Teleport cluster without using any shared secrets as with the`token` join
+method. Instead of using a shared secret, the unique identity of the host's
```

Comment:
I would use active voice in this sentence to indicate which component is doing the authenticating. Which Teleport cluster components are involved in this authentication flow?

Reply:
258e78d2b9